internal/xsql/conn.go

```diff
@@ -377,56 +377,80 @@ func (c *conn) ResetSession(_ context.Context) error {
 	return nil
 }
 
-func (c *conn) MetadataQuery(ctx context.Context, query string, args []driver.NamedValue) (_ driver.Rows, err error) {
-	const (
-		metadataQuery          = "metadata-query"
-		versionQueryMode       = "version"
-		isTableExistQueryMode  = "is-table-exist"
-		isColumnExistQueryMode = "is-column-exist"
-		getColumnsQueryMode    = "get-columns"
-		getTablesQueryMode     = "get-tables"
-		getIndexesQueryMode    = "get-indexes"
-	)
-
+func (c *conn) Version(ctx context.Context) (_ driver.Rows, err error) {
```

Comment:
Return type - string (1) or strings (2) (major, minor, patch) and error
(1) is simple and better

Reply:
because at now YDB cannot return self version. 
Now need to return "default" for example

Reply:
You need to know next info: YDB is a distributed RDBMs and version - is a versions of all nodes YDB. 
For example some nodes have 22-4 version, some - 22-2, some - 22-5. And its O'k